app/api/items/create_item.go

```diff
@@ -27,6 +28,7 @@ type Item struct {
 	EntryFrozenTeams bool `json:"entry_frozen_teams"`
 	// default: false
 	NoScore bool `json:"no_score"`
+	// Unique identifier to reference the task. An error is returned if another task has the same `text_id`
```

Comment:
Shouldn't the doc "explain" how to spot this error in the output and what will be the error code in such a case (400? 403?) ? Or do we consider the way as standard ? 
Could be either in a specific 403 output (more "logic" place) or the input text should state that the error will be a 403 with `text_id` key in `errors` (more convenient).

Reply:
It's the same as any other field related error: 403 with a key for the field, and the error, along. I changed the doc to just mark the field as "Unique" instead of mentionning the error. To make it the same as when "Required" is mentionned for example.